R/fimsframe.R

```diff
@@ -54,105 +46,110 @@ setMethod("get_data", "FIMSFrame", function(x) x@data)
 setGeneric("fleets", function(x) standardGeneric("fleets"))
 setMethod("fleets", "FIMSFrame", function(x) x@fleets)
 
-setGeneric("nyrs", function(x) standardGeneric("nyrs"))
-setMethod("nyrs", "FIMSFrame", function(x) x@nyrs)
+setGeneric("n_years", function(x) standardGeneric("n_years"))
+setMethod("n_years", "FIMSFrame", function(x) x@n_years)
 
 setGeneric("start_year", function(x) standardGeneric("start_year"))
 setMethod("start_year", "FIMSFrame", function(x) x@start_year)
 
 setGeneric("end_year", function(x) standardGeneric("end_year"))
 setMethod("end_year", "FIMSFrame", function(x) x@end_year)
 
-# additional accessors for FIMSFrameAge
 setGeneric("ages", function(x) standardGeneric("ages"))
-setMethod("ages", "FIMSFrameAge", function(x) x@ages)
+setMethod("ages", "FIMSFrame", function(x) x@ages)
 
-setGeneric("nages", function(x) standardGeneric("nages"))
-setMethod("nages", "FIMSFrameAge", function(x) x@nages)
+setGeneric("n_ages", function(x) standardGeneric("n_ages"))
+setMethod("n_ages", "FIMSFrame", function(x) x@n_ages)
 
-setGeneric("weightatage", function(x) standardGeneric("weightatage"))
-setMethod("weightatage", "FIMSFrameAge", function(x) x@weightatage)
+setGeneric("weight_at_age", function(x) standardGeneric("weight_at_age"))
+setMethod("weight_at_age", "FIMSFrame", function(x) x@weight_at_age)
 
-setGeneric("m_weightatage", function(x) standardGeneric("m_weightatage"))
+setGeneric("m_weight_at_age", function(x) standardGeneric("m_weight_at_age"))
```

Comment:
Question: I remember m_*() means module, do we want to explicitly write it out in the code (e.g., module_weight_at_age) or add a line of documentation in this file where it was initially introduced?

Reply:
I think we should leave it as is until we decide how to "hide" these `m_*()` functions and just make them happen internally rather than how they are being used now within the case studies.